src/debug/debugSupport.ts

```diff
@@ -45,3 +51,86 @@ export async function generateSourceMapUrl(functionText: string, generatedText:
   const sourceMapUrl = await sourceMap.generateSourceMapUrl(functionText, generatedText);
   return sourceMapUrl || generateSourceUrl();
 }
+
+export async function installConsoleHelpers(context: BrowserContextBase) {
+  if (!isDebugMode())
+    return;
+  const installInFrame = async (frame: Frame) => {
+    try {
+      const main = await frame._mainContext();
+      const injectedScript = await main.injectedScript();
+      await injectedScript.evaluate((injectedScript, parseSelectorString) => {
+        const parseSelector: (selector: string) => types.ParsedSelector = new Function('...args', 'return (' + parseSelectorString + ')(...args)') as any;
+        const checkSelector = (parsed: types.ParsedSelector) => {
+          for (const {name} of parsed.parts) {
+            if (!injectedScript.engines.has(name))
+              throw new Error(`Unknown engine "${name}"`);
+          }
+        };
+
+        const glassPane = document.createElement('div');
+        glassPane.style.cssText = 'position: absolute; left: 0; top: 0; pointer-events: none; overflow: visible; z-index: 10000;';
```

Comment:
How do you use this glass pane?

Reply:
It is zero-by-zero at the top-left, only used for positioning the highlights. Let me rename it to container.